src/api-service/__app__/onefuzzlib/task_event.py

```diff
@@ -13,10 +13,19 @@
 
 
 class TaskEvent(BASE_TASK_EVENT, ORMMixin):
+    # handle sorting TaskEvents even if one or more of Timestamps are not set yet
+    def __lt__(self, other: "TaskEvent") -> bool:
+        if self.Timestamp is None:
+            return False
```

Comment:
Why are we handling `None` asymmetrically?

Reply:
Clarifying: we seem to be treating `None` as `+infinity`, sort of, so we can always compare `TaskEvent` instances, even if one or both have a `None` timestamp.

Why is this choice the right one? What about if we have multiple task event instances with `None` timestamps?

If it's actually always an error to have a `None` timestamp, should we throw an error instead of just picking some convention?

Reply:
This convention makes it such that the sorting is stable if we don't have a Timestamp and doesn't cause an Exception in the case we don't have Timestamp.